### PR TITLE
Find donations conflicts for givbacks

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
@CarlosQ96 
I have deleted src/delete-uniswap/tokenLists.ts  list to just have one list for tokens list
( the findConflict script doesn't have any side effect on our application, you just should check the token list is ok)

### CODE REVIEW

File: src/scripts/findConflicts.ts [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+
+const EXCLUDED_FROM_ADDRESSES = ['0x0000000000000000000000000000000000000000'];
+
+const smartContractAddresses: string[] = [];
+
+// 2021/12/24 16:00:00 GMT, beginning of round
+const beginTimestamp = 1640361600;
+
+// 2021/01/07 16:00:00 GMT, end of round
+const endTimestamp = 1641571200;
```
Comment (CarlosQ96) [inline src/scripts/findConflicts.ts:34]: Is the script reusable? Shouldn't we move this variables to env vars, so we only have to change them and rerun the script when required?

Comment (CarlosQ96) [reply src/scripts/findConflicts.ts:34]: Maybe for future rounds.

**Review by CarlosQ96 — APPROVED** [review]
Aside from that I think the token list is fine and I am sure you tested with Griff and the team the script after looking at the chats.

File: src/scripts/findConflicts.ts (hunk 1 above)
Comment (mohammadranjbarz) [reply src/scripts/findConflicts.ts:34]: You know when changing it for future round it's not needed to commit anything, I think we can merge it now, in the future we can change it
Thanks for your suggestion

**Pull request merged**